cl/recap/mergers.py

```diff
@@ -1,4 +1,5 @@
 # Code for merging PACER content into the DB
+import asyncio
```

Comment:
I think now this is not required, right?

Reply:
yeah, removed